swagger/swagger.go

```diff
@@ -85,16 +91,20 @@ func New(config ...Config) fiber.Handler {
 		}
 	}
 
-	// Verify Swagger file exists
-	if _, err := os.Stat(cfg.FilePath); os.IsNotExist(err) {
-		panic(fmt.Errorf("%s file does not exist", cfg.FilePath))
-	}
+	rawSpec := cfg.FileContent
```

Comment:
This needs to be inside the IF above where cfg is override

Reply:
I don't follow. rawSpec is the data to use. If FileContent is set, then we use that, otherwise we load it from FilePath in the following if-statement.

While looking at the above if-statement, I realized that cfg.FilePath should only be set from DefaultConfig if cfg.FileContent is also empty, so I added that in [75a50b6](https://github.com/gofiber/contrib/pull/1085/commits/75a50b64281d49fc38f51810749967444941eb44).

Reply:
>While looking at the above if-statement, I realized that cfg.FilePath should only be set from DefaultConfig if cfg.FileContent is also empty, so I added that in [75a50b6](https://github.com/gofiber/contrib/pull/1085/commits/75a50b64281d49fc38f51810749967444941eb44).

This was wrong. cfg.FilePath still needs to be set like before for serving /swagger.json. Force pushed to remove the fixup commit.

Reply:
Hmmm, it wouldnt use FileContent then. My suspicion is that the tests are using the file not the content since they both reference "swagger.json". We need a new file to compare properly

Reply:
>Hmmm, it wouldnt use FileContent then.

What do you mean? It always uses FileContent if its length is > 0. Here is the full context:

```go
	rawSpec := cfg.FileContent
	if len(rawSpec) == 0 {
		// Verify Swagger file exists
		_, err := os.Stat(cfg.FilePath)
		if os.IsNotExist(err) {
			panic(fmt.Errorf("%s file does not exist", cfg.FilePath))
		}

		// Read Swagger Spec into memory
		rawSpec, err = os.ReadFile(cfg.FilePath)
		if err != nil {
			log.Fatalf("Failed to read provided Swagger file (%s): %v", cfg.FilePath, err.Error())
			panic(err)
		}
	}
```

>My suspicion is that the tests are using the file not the content since they both reference "swagger.json". We need a new file to compare properly

I'll change the FilePath in the tests to avoid the potential overlap.

Reply:
[ac66002](https://github.com/gofiber/contrib/pull/1085/commits/ac66002a48f4eddedd2e70129e1dbba524d04196)